torch/_dynamo/variables/builtin.py

```diff
@@ -358,7 +358,39 @@ def call_function(
             )
         return super().call_function(tx, args, kwargs)
 
-    def _call_min_max(self, tx, a, b):
+    def _call_min_max(self, tx, *args):
+        if all(isinstance(a, variables.ConstantVariable) for a in args):
+            if self.fn is max:
+                return variables.ConstantVariable(max(a.value for a in args))
+            else:
+                return variables.ConstantVariable(min(a.value, b.value))
+
+        # torch min/max only support two arguments, but python min/max support a variable number.
```

Comment:
Every time I make a change in dynamo, I question whether or not the added complexity is because I'm doing it wrong, or of something fundamental in the dynamo -> torch translation.

This one feels more fundamental, but pls tell me if otherwise (cc @voznesenskym)